tests/functional/policies/check_triggers/w02.bad_if.if

```diff
@@ -0,0 +1,37 @@
+## <summary>dummy.if</summary>
+
+#######################################
+## <summary>
+##      A template
+## </summary>
+## <param name="dummy_prefix">
+##      <summary>
+##      The prefix of the dummy (e.g., foo
+##      is the prefix for foo_t).
+##      </summary>
+## </param>
+#
+template(`dummy_template',`
+        gen_require(`
+                type etc_t;
+        ')
+
+        type $1_t;
+        domain_type($1_t) # Generates a W-002
```

Comment:
This (and also the next) comment is a bit misleading, since the test runner `test_one_check_expect "W-002" "w02.bad_if.if" 0` expect zero warnings.

Reply:
Thanks!  Yeah, I copied the file the issue reported supplied for the test and it had his comments from when they were reporting those warnings.  Should be fixed now.